src/Microsoft.HttpRepl/Commands/HelpCommand.cs

```diff
@@ -46,7 +47,30 @@ public async Task ExecuteAsync(IShellState shellState, HttpState programState, I
                             if (!string.IsNullOrEmpty(help))
                             {
                                 anyHelp = true;
+                                shellState.ConsoleManager.WriteLine();
                                 shellState.ConsoleManager.WriteLine(help);
+
+                                var structuredCommand = command as CommandWithStructuredInputBase<HttpState, ICoreParseResult>;
```

Comment:
This is a reasonable way of going about this. I'm not a huge fan of this only working for things that derive from CommandWithStructuredInputBase<HttpState, ICoreParseResult>, but I don't see us adding additional commands that don't derive from that - making InputSpec public doesn't feel the best either, but I don't think it hurts anything.

We could add a method to get the options help to ICommand<,> and implement it in CommandWithStructuredInputBase<,> with the logic below - this would allow InputSpec to become protected again and lift the base type restriction, but it probably doesn't matter too much.